### PR TITLE
more style fixes in easyconfigs

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86 please review?

File: easybuild/easyconfigs/s/STAR/STAR-2.5.0a-GNU-4.9.3-2.25.eb [hunk 1]
```diff
@@ -29,7 +28,7 @@ buildopts = ' STAR && make STARlong'
 parallel = 1
 
 files_to_copy = [(["source/STAR", "source/STARlong"], "bin"), "CHANGES.md", "doc", "extras", "LICENSE", "README.md", "RELEASEnotes.md"]
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/s/STAR/STAR-2.5.0a-GNU-4.9.3-2.25.eb]: line too long (135 > 120 characters)

File: easybuild/easyconfigs/s/STAR/STAR-2.5.0a-goolf-1.4.10.eb [hunk 2]
```diff
@@ -29,7 +28,7 @@ buildopts = ' STAR && make STARlong'
 parallel = 1
 
 files_to_copy = [(["source/STAR", "source/STARlong"], "bin"), "CHANGES.md", "doc", "extras", "LICENSE", "README.md", "RELEASEnotes.md"]
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/s/STAR/STAR-2.5.0a-goolf-1.4.10.eb]: line too long (135 > 120 characters)

File: easybuild/easyconfigs/s/STAR/STAR-2.5.1b-foss-2015b.eb [hunk 3]
```diff
@@ -31,7 +30,7 @@ buildopts = ' STAR && make STARlong'
 parallel = 1
 
 files_to_copy = [(["source/STAR", "source/STARlong"], "bin"), "CHANGES.md", "doc", "extras", "LICENSE", "README.md", "RELEASEnotes.md"]
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/s/STAR/STAR-2.5.1b-foss-2015b.eb]: line too long (135 > 120 characters)

File: easybuild/easyconfigs/s/STAR/STAR-2.5.1b-goolf-1.4.10.eb [hunk 4]
```diff
@@ -29,7 +28,7 @@ buildopts = ' STAR && make STARlong'
 parallel = 1
 
 files_to_copy = [(["source/STAR", "source/STARlong"], "bin"), "CHANGES.md", "doc", "extras", "LICENSE", "README.md", "RELEASEnotes.md"]
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/s/STAR/STAR-2.5.1b-goolf-1.4.10.eb]: line too long (135 > 120 characters)

File: easybuild/easyconfigs/s/STAR/STAR-2.5.2a-foss-2016a.eb [hunk 5]
```diff
@@ -32,15 +31,15 @@ buildopts = ' STAR && make STARlong'
...
 
 files_to_copy = [
-    (["source/STAR", "source/STARlong"], "bin"), 
-      "CHANGES.md", 
-      "doc", 
-      "extras", 
-      "LICENSE", 
-      "README.md", 
+    (["source/STAR", "source/STARlong"], "bin"),
+      "CHANGES.md",
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/s/STAR/STAR-2.5.2a-foss-2016a.eb]: continuation line unaligned for hanging indent

File: easybuild/easyconfigs/s/STAR/STAR-2.5.2a-goolf-1.7.20.eb [hunk 6]
```diff
@@ -29,15 +28,15 @@ buildopts = ' STAR && make STARlong'
...
 
 files_to_copy = [
-    (["source/STAR", "source/STARlong"], "bin"), 
-      "CHANGES.md", 
-      "doc", 
-      "extras", 
-      "LICENSE", 
-      "README.md", 
+    (["source/STAR", "source/STARlong"], "bin"),
+      "CHANGES.md",
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/s/STAR/STAR-2.5.2a-goolf-1.7.20.eb]: continuation line unaligned for hanging indent

File: easybuild/easyconfigs/s/Serf/Serf-1.3.8-foss-2015a.eb [hunk 7]
```diff
@@ -21,8 +21,8 @@ builddependencies = [('SCons', '2.3.6', '-Python-2.7.9')]
 dependencies = [
     ('APR', '1.5.2'),
     ('APR-util', '1.5.4'),
-    #   ('OpenSSL', '1.0.1p'),  # OS dependency should be preferred if the os version is more recent then this version, it's
-    #   nice to have an up to date openssl for security reasons
+    # ('OpenSSL', '1.0.1p'),  # OS dependency should be preferred if the os version is more recent then this version, it's
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/s/Serf/Serf-1.3.8-foss-2015a.eb]: line too long (122 > 120 characters)

File: easybuild/easyconfigs/s/STAR/STAR-2.5.0a-GNU-4.9.3-2.25.eb (hunk 1 above)
Comment (wpoely86) [inline easybuild/easyconfigs/s/STAR/STAR-2.5.0a-GNU-4.9.3-2.25.eb]: Lines that are too long should be picked up by the tests?

Comment (boegel) [inline easybuild/easyconfigs/s/STAR/STAR-2.5.0a-GNU-4.9.3-2.25.eb]: @wpoely86 The style tests are not enabled yet, we need to resolve over 1k style issues in the current easyconfigs collection before we can do that...

I'm gradually working towards making `--style-check` pass on all the easyconfigs we have now, with the intention to enable the style checks ASAP

Comment (wpoely86) [inline easybuild/easyconfigs/s/STAR/STAR-2.5.0a-GNU-4.9.3-2.25.eb]: @boegel Okay, but please run them at least on all the once you change?

Comment (boegel) [inline easybuild/easyconfigs/s/STAR/STAR-2.5.0a-GNU-4.9.3-2.25.eb]: Yeah, good point, I should

**Comment by boegel** [comment]
Thanks for the review @wpoely86 and @migueldiascosta!

**Pull request merged**